### ISSUE TITLE
java.lang.AbstractMethodError and dependency mess

### ISSUE THREAD

**Issue by RokLenarcic** [metosin/compojure-api#476]
## Library Version(s)

2.0.0-alpha30

## Problem

I get this error while trying to display the UI:
```
java.lang.AbstractMethodError: Receiver class linked.map.LinkedMap does not define or inherit an implementation of the resolved method 'abstract boolean isEmpty()' of interface java.util.Map.
```

This happens due to a bug in `frankiesardo/linked` dependency. You use `ikitommi/linked` but you also include `frankiesardo/linked` via `metosin/ring-swagger`. This error happens for some people but not for others, depending on the order that JVM grabs these two dependencies.

**Comment by RokLenarcic** [comment]
I need to upgrade this dep.

**Comment by RokLenarcic** [comment]
I see your upgrade of `ring-swagger` from 0.26.2 to 1.0.0 doesn't upgrade/remove `frankiesardo/linked` at all. So this bug still stands.

**Comment by frenchy64** [comment]
Thanks for the report, that's concerning.

> I need to upgrade this dep.

What do you recommend specifically? What does "upgrade" mean here?

**Comment by RokLenarcic** [comment]
I tested with "not the latest" version at that point so I closed the issue because I saw that 1.0.0 version has different dependencies. Tried with that version and the problem persists, so I reopened. Solved it on my end by explicitly  adding exclusion for frankiesardo/linked.
